packages/cryptokit/cryptokit.1.17/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+synopsis: "A library of cryptographic primitives"
+description:
+  "Cryptokit includes authenticated encryption (AES-GCM, Chacha20-Poly1305), block ciphers (AES, DES, 3DES), stream ciphers (Chacha20, ARCfour), public-key cryptography (RSA, DH), hashes (SHA-256, SHA-512, SHA-3, Blake2), MACs, compression, random number generation -- all presented with a compositional, extensible interface."
+license: "GNU Library General Public License v2 or later with OCaml LGPL Linking Exception"
```

Comment:
```suggestion
license: "LGPL-2.0-or-later WITH OCaml-LGPL-linking-exception"
```

Reply:
Makes sense, thanks.  Before writing the license field I searched the Web for documentation on the license field in OPAM files and didn't find anything.

Reply:
You are perfectly right, thanks for pointing it out. I have opened an issue on the opam repository to improve the license field description and include an example with the `WITH` keyword.